msteams-platform/sbs-gs-workflow-bot.yml

```diff
@@ -23,7 +23,7 @@ items:
     
```

Comment:
The wording in the statement is confusing. It is unclear whether it refers to customizing the installation of the bot app or customizing the Adaptive Card response of the bot app. However, neither of these aspects are demonstrated in this tutorial.

Reply:
Updated.